Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -2274,6 +2282,100 @@ - (void)writingToolsSession:(WTSession *)session didReceiveAction:(WTAction)acti
     _page->writingToolsSessionDidReceiveAction(*webSession, webAction);
 }
 
+#pragma mark - WKIntelligenceTextEffectCoordinatorDelegate conformance
+
+// FIXME: Give the `coordinator` parameters actual types.
```

Comment:
Just curious — is there anything blocking this FIXME from being resolved? (Or is it going to be fixed in a subsequent patch)